Workflow/Sources/WorkflowLogger.swift

```diff
@@ -14,10 +14,64 @@
  * limitations under the License.
  */
 
-import os.signpost
+import OSLog
 
 private extension OSLog {
+    /// Logging will use this log handle when enabled
     static let workflow = OSLog(subsystem: "com.squareup.Workflow", category: "Workflow")
+
+    /// The active log handle to use when logging. Defaults to the shared `.disabled` handle.
+    static var active: OSLog = .disabled
```

Comment:
Just for my own clarification.  This defaulting to `.disabled` is why nothing is logged by default?  You would have to changed this to `workflow` from line 21 for the log to be active?

Reply:
that's right, this is the change that effectively turns off logging by default. consumers would have to do (at a minimum) `WorkflowLogging.enabled = true` to allow anything to be logged. i can add some additional docs to the files clarifying this

Reply:
👍  Looks fine to me.